src/elexmodel/cli.py

```diff
@@ -39,6 +39,7 @@
     type=click.Choice(["county", "precinct", "county-district", "precinct-district"]),
 )
 @click.option("--beta", "beta", default=1, type=int, help="manually add variance to Gaussian model")
+@click.option("--winsorize", "winsorize", default=False, type=bool, help="reduce outliers in the Gaussian model")
```

Comment:
I think we want to make this a flag (rather than an explicit parameter). Take a look at what robust looks like just below. The difference is:
```
elexmodel 2021-01-05_GA_G --office_id=S_precinct --estimands=dem --geographic_unit_type=precinct --pi_method=gaussian --percent_reporting=5 --winsorize=True
```
vs. 
```
elexmodel 2021-01-05_GA_G --office_id=S_precinct --estimands=dem --geographic_unit_type=precinct --pi_method=gaussian --percent_reporting=5 --winsorize
```